### PR TITLE
Traduccion login

### PR DESCRIPTION
**Pull request by r4ulill0**

### CODE REVIEW

File: Surveyx/app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -18,6 +18,11 @@
     <string name="question_mark">\?</string>
 
     <!-- LOGIN -->
+    <string name="login_user_placeholder">Nombre de usuario</string>
+    <string name="login_pass_placeholder">Contaseña</string>
```
Comment (jvillafanez) [inline Surveyx/app/src/main/res/values-es/strings.xml]: "Contaseña"?

**Comment by pokevik21** [comment]
la variable del "remember me" se llama "remember_me" en el fichero de recurso, 
Surveyx/app/src/main/res/values-es/strings.xml

File: Surveyx/app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
+    <string name="login_button">Iniciar sesión</string>
+    <string name="incorrect_login">Inicio de sesión incorrecto</string>
+    <string name="remember_me">Recuerdame</string>
```
Comment (jrodgut-sg) [inline Surveyx/app/src/main/res/values-es/strings.xml]: Recuérdame

File: Surveyx/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -18,14 +18,13 @@
...
-    <string name="login_pass_placeholder"></string>
-    <string name="login_button"></string>
-    <string name="incorrect_login"></string>
-    <string name="descriptor_login_user"></string>
-    <string name="descriptor_login_pass"></string>
-    <string name="descriptor_login_button"></string>
+    <string name="login_title">SOLIDGEAR ACADEMY</string>
+    <string name="login_user_placeholder">username</string>
+    <string name="login_pass_placeholder">password</string>
+    <string name="login_button">sing in</string>
```
Comment (jvillafanez) [inline Surveyx/app/src/main/res/values/strings.xml]: Case sensitiveness mismatch regardless Spanish translation

File: Surveyx/app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -18,6 +18,11 @@
     <string name="question_mark">\?</string>
 
     <!-- LOGIN -->
+    <string name="login_user_placeholder">Nombre de usuario</string>
```
Comment (jvillafanez) [inline Surveyx/app/src/main/res/values-es/strings.xml]: Missing "login_title" string... intended?

File: Surveyx/app/src/main/res/values-es/strings.xml (hunk 2 above)
Comment (ismtabo) [inline Surveyx/app/src/main/res/values-es/strings.xml]: Spelling error at "Recuérdame".

File: Surveyx/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (ismtabo) [inline Surveyx/app/src/main/res/values/strings.xml]: Should not the login button's text start with uppercase?

File: Surveyx/app/src/main/res/values-es/strings.xml (hunk 4 above)
Comment (r4ulill0) [reply Surveyx/app/src/main/res/values-es/strings.xml]: Yes, it doesn't need a translation. "Solid Gear Academia" sounds weird.

File: Surveyx/app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -18,14 +18,13 @@
...
-    <string name="login_pass_placeholder"></string>
-    <string name="login_button"></string>
-    <string name="incorrect_login"></string>
-    <string name="descriptor_login_user"></string>
-    <string name="descriptor_login_pass"></string>
-    <string name="descriptor_login_button"></string>
+    <string name="login_title">SOLIDGEAR ACADEMY</string>
+    <string name="login_user_placeholder">Username</string>
+    <string name="login_pass_placeholder">Password</string>
+    <string name="login_button">Sing in</string>
```
Comment (jvillafanez) [inline Surveyx/app/src/main/res/values/strings.xml]: ❓ 
sing pig

File: Surveyx/app/src/main/res/values-es/strings.xml (hunk 4 above)
Comment (jvillafanez) [reply Surveyx/app/src/main/res/values-es/strings.xml]: 👍

File: Surveyx/app/src/main/res/values-es/strings.xml (hunk 2 above)
Comment (r4ulill0) [reply Surveyx/app/src/main/res/values-es/strings.xml]: @jrodgut-sg fixed

Comment (r4ulill0) [reply Surveyx/app/src/main/res/values-es/strings.xml]: @ismtabo fixed

File: Surveyx/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (r4ulill0) [reply Surveyx/app/src/main/res/values/strings.xml]: @jvillafanez done

File: Surveyx/app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (r4ulill0) [reply Surveyx/app/src/main/res/values-es/strings.xml]: @jvillafanez fixed, thank you.

File: Surveyx/app/src/main/res/values/strings.xml (hunk 5 above)
Comment (r4ulill0) [reply Surveyx/app/src/main/res/values/strings.xml]: @jvillafanez fixed

**Pull request merged**